v1Tov2Migrations/helper/createCapture.js

```diff
@@ -51,8 +51,8 @@ const createCapture = async (rawCapture, tree, trx, treeTags) => {
     note: rawCapture.note,
     attributes: rawCapture.extra_attributes,
     domain_specific_data: tree.domain_specific_data,
-    created_at: new Date().toISOString(),
-    updated_at: new Date().toISOString(),
+    created_at: rawCapture.captured_at,
+    updated_at: rawCapture.captured_at,
```

Comment:
For the migration script, using the captured_at rather then the time of running this script makes more sense.

Reply:
@dadiorchen or should we just add a new ```captured_at``` field to the capture table?

Reply:
I agree. Please add it.